packages/client/tests/functional/interactive-transactions/__snapshots__/tests.ts.snap

```diff
@@ -0,0 +1,121 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`interactive-transactions (mongodb) () batching rollback 1`] = `
```

Comment:
All these snapshots have different output per provider, so I had to convert inline snapshots into external ones.